packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationModule.java

```diff
@@ -96,7 +100,7 @@ public void isEnrolledAsync(final Promise promise) {
   }
 
   @ExpoMethod
-  public void authenticateAsync(final Promise promise) {
+  public void authenticateAsync(final ReadableArguments options, final Promise promise) {
```

Comment:
```suggestion
  public void authenticateAsync(final Map<String, Object> options, final Promise promise) {
```
In the expo, we use `Map` rather than `ReadableArguments`.